content/en/users/datatransfer/api/_index.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "API"
+linkTitle: "API"
+type: docs
+description: "Documentation for EGI Data Transfer API"
+weight: 30
+---
+
+## Overview 
+
+The EGI Data Transfer service offers API both for Users and Admins, in this section 
+we are focusing on the User API.
+
+2 APIs are available to users:
+
+* RESTFul API
```

Comment:
Before you have been using `-` instead of `*`.
Please also removed unneeded empty lines/double empty lines.